README.md

```diff
@@ -1,8 +1,10 @@
 # Rack::ContentTypeDefault
 
 Rack::ContentTypeDefault is a tiny piece of middleware that sets default `CONTENT-TYPE:` header when it isn't present
-for specific methods. It checks if a content type is set and does not override the existing one.
-The options allow you to specify the request methods, and the content type to be set.
+for specific requests. It checks if a content type is set and does not override the existing one.
+The options allow you to specify the request methods and the content type to be set. The options also allow you to set
```

Comment:
minor: How about a line break here. and simply list out all possible options as a numbered or bulleted list. Something like:

The following options may be specified:
* Request methods
* Desired content type
* Paths to which the content type is desired/applicable. `all` being the default when not specified.
* Another content type option that only applies to `application/xml` or `application/json` if the path ends in .xml or .json respectively.